packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -95,6 +95,46 @@ export const tide = (axios, site, config) => ({
     return axios.$post(url, data, axiosConfig)
   },
 
+  delete: async function (url, headersConfig = {}) {
```

Comment:
This should be removed as the front end should not be deleting the files from Drupal.

Reply:
Hey @dylankelly, this has been removed now. Thanks